client/src/js/services/account.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+/**
+ * @description 
+ *
+ * @returns 
+ */
+
+angular.module('bhima.services')
+  .factory('Account', Account);
+  
+Account.$inject = ['$http'];
+
+function Account($http) {  
+  
+  function list() { 
+    var path = '/accounts?list=\'full\'';
```

Comment:
You don't need the `\'` marks.  You can just write:
```js
var path = '/accounts?list=full';
```
and ExpressJS will transform "full" into a string.

A better way to express this is not a string, but a boolean!  Try `/accounts?full=1` or `/accounts?full=0` instead to avoid string comparison in the server controller and shorten your URL.